packages/react/src/i18n/translations.ts

```diff
@@ -45,7 +45,8 @@ export const Translations = {
             ariaLabel: 'Menu',
         },
         pagination: {
-            results: 'results',
+            pagination: 'Pagination',
+            results: '{{pageStartIndex}}-{{pageEndIndex}} of {{numberOfResults}} results',
```

Comment:
Ce n'est pas specifie dans JIRA, mais on devrait plutot utiliser le "En dash" au lieu du tiret lorsque c'est un range.

https://www.thepunctuationguide.com/en-dash.html#:~:text=The%20en%20dash%20is%20used,season%20was%20our%20best%20yet.